doc/dev/find_sdk_in_cli_command.md

```diff
@@ -0,0 +1,88 @@
+```python
+def create_resource_group(cmd, rg_name, location, tags=None, managed_by=None):
+    """ Create a new resource group.
+    :param str resource_group_name:the desired resource group name
+    :param str location:the resource group location
+    :param str tags:tags in 'a=b c' format
+    """
+    rcf = _resource_client_factory(cmd.cli_ctx)
+
+    ResourceGroup = cmd.get_models('ResourceGroup')
+    parameters = ResourceGroup(
+        location=location,
+        tags=tags
+    )
+
+    if cmd.supported_api_version(min_api='2016-09-01'):
+        parameters.managed_by = managed_by
+
+    return rcf.resource_groups.create_or_update(rg_name, parameters)
+```
+Then, you could find the `create_or_update()` function in its return, it is the SDK function.
+
+## Find the function in Python SDK or Doc
+
+Similarly, find the `service` folder in the [SDK folder](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk) of [SDK repo](https://github.com/Azure/azure-sdk-for-python).
+Such as [`resources` folder](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/resources).
+
+Then, you could find [here](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources). This step may be a little difficult for beginners. It requires some experience.
+Don't be frightened by a lot of folders here. You can choose the latest date to enter, like [this](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources/v2021_04_01).
+
+Enter the [_operation.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources/v2021_04_01/operations/_operations.py) in the [operation folder](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources/v2021_04_01/operations), and you can see the meta code.
+
+In this file, you could search the function name to find the function definition.(Be careful, the function need under the correct class.)
+Above the `begin_delete` function, you could find it from [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources/v2021_04_01/operations/_operations.py#L8784).
+The same, the `create_or_update` function is [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources/v2021_04_01/operations/_operations.py#L9761).
+
+Finally, you also could find the usage of function by Azure doc, like [this](https://docs.microsoft.com/en-us/azure/developer/python/sdk/examples/azure-sdk-example-resource-group#3-write-code-to-provision-a-resource-group).
+
+Feel free to contact Azure SDK team at any time through any channels. We are passionate to build the world-class cloud product.
```

Comment:
if customers have found package name and function, they need sample about how to use the package. maybe we could guide them to https://github.com/Azure-Samples/azure-samples-python-management